tasks.rb

```diff
@@ -2424,28 +2431,42 @@ def set_hpxml_heating_systems(hpxml_file, hpxml)
          'base-hvac-ideal-air.xml',
          'base-hvac-none.xml',
          'base-hvac-room-ac-only.xml',
+         'base-hvac-shared-chiller-only-baseboard.xml',
          'invalid_files/orphaned-hvac-distribution.xml'].include? hpxml_file
     hpxml.heating_systems.clear
   elsif ['base-hvac-boiler-elec-only.xml'].include? hpxml_file
     hpxml.heating_systems[0].heating_system_type = HPXML::HVACTypeBoiler
     hpxml.heating_systems[0].heating_system_fuel = HPXML::FuelTypeElectricity
     hpxml.heating_systems[0].heating_efficiency_afue = 1
+    hpxml.heating_systems[0].is_shared_system = false
```

Comment:
Oh I just realized is_shared_system is required to explicitly specify as "true" or "false" here. I was treating not specifying "is_shared_system" as "is_shared_system" = false in shared mech vent.

Reply:
Yes I defined it as a boolean in the HPXML schema, so we should use boolean in our HPXML class here too.